### ISSUE TITLE
JAWS 2025 does not announce TinyMCE label when focus is brought to editable area

### ISSUE THREAD

**Issue by pastr1** [FreedomScientific/VFO-standards-support#902]
### Summary
When bringing focus into the editable area of the TinyMCE rich text editor (RTE) field, the aria-label value is not announced by JAWS 2025.

![Image](https://github.com/user-attachments/assets/d4b128cb-347e-4873-ba58-9de120bfe96f)

### Steps to reproduce
Reproducible in TinyMCE's Basic RTE example
https://www.tiny.cloud/docs/tinymce/latest/basic-example/

With JAWS 2024 or JAWS 2025 enabled, bring focus into the editable area of the RTE using the Tab key. Only "Type in text" is announced, as opposed to the aria-label value of "Rich Text Area. Press ALT-0 for help. (edit type in text)"

### Expected result
JAWS announces the label of the RTE when focus is brought into the field.

### JAWS version and build number
JAWS 2025.2412.50

### Additional comments
This appears to be a regression from JAWS 2022 as the aria-label is announced when bringing focus into the RTE with v. 2022:

![Image](https://github.com/user-attachments/assets/438cde19-7e48-4f53-88f4-0928a011982d)

When using NVDA, the issue does not occur:

![Image](https://github.com/user-attachments/assets/c13c1441-f6b9-4ca8-bdf2-3ea6376f6156)

If the _role="application"_ attribute is removed from the <div class="tox tox-tinymce tox-edit-focus"... HTML tag, JAWS 2025 announces the aria-label value as expected when focus is brought into the RTE's editable area:

![Image](https://github.com/user-attachments/assets/5398308c-e946-4588-9d2d-b82051ea9559)

**Comment by JeremiahRogersVispero** [comment]
Appears to work fine using J2025.2506.170.
